issue-templates/forms/general/bug-report.yml

```diff
@@ -0,0 +1,58 @@
+# Source: https://github.com/arduino/tooling-project-assets/blob/main/issue-templates/forms/platform-dependent/bug-report.md
+# See: https://docs.github.com/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms
+
+name: Bug report
+description: Report a problem with the code or documentation in this repository.
+labels:
+  - "type: imperfection"
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the problem
+    validations:
+      required: true
+  - type: textarea
+    id: reproduce
+    attributes:
+      label: To reproduce
+      description: Provide the specific set of steps we can follow to reproduce the problem.
+    validations:
+      required: true
+  - type: textarea
+    id: expected
+    attributes:
+      label: Expected behavior
+      description: What would you expect to happen after following those instructions?
+    validations:
+      required: true
+  - type: input
+    id: project-version
+    attributes:
+      # TODO: Replace TODO_PRODUCT_NAME with the product name of the project.
+      label: TODO_PRODUCT_NAME version
+      description: |
+        Which version of TODO_PRODUCT_NAME are you using?
+        *This should be the most recent version available.*
```

Comment:
I think we should communicate more clearly to the user that he should first try if the bug is present in the latest version

Reply:
Do you have any suggestions?

You can see an approximate preview of how these forms look here:

https://github.com/arduino/arduino-ide/issues/new?assignees=&labels=type%3A+imperfection&template=bug-report.yml

The significant difference is that there isn't a "nightly build" link in these general purpose forms because that won't be available in many of the projects.

Note that there is a mandatory checkbox at the bottom of the form:

![image](https://user-images.githubusercontent.com/8572152/167408848-e165c99d-311e-47dc-8a13-ee7e3fd19613.png)

I'm also planning on producing some reusable CONTRIBUTING.md files, which will also provide instructions to test against the latest version, something along the lines of this:

https://arduino.github.io/arduino-lint/dev/CONTRIBUTING/#issue-reports

GitHub encourages the contributor to read CONTRIBUTING.md during the issue creation process, so I think that is the best place to put detailed instructions for the contributor..